scripts/pylint_custom_plugin/pylint_guidelines_checker.py

```diff
@@ -1706,6 +1706,68 @@ def visit_functiondef(self, node):
     visit_asyncfunctiondef = visit_functiondef
 
 
+class CheckNamingMismatch(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = "check-naming"
```

Comment:
What is this name used for? And should it be more explicit? Like `check-naming-mismatch`